applications/Tenure_and_Resource_Stewardship_Branch.md

```diff
@@ -0,0 +1,151 @@
+
+**Specifications** are documents containing any combination of Machine-Testable Requirements and Human-Auditable Requirements needed to produce technical interoperability.
+
+## 1.3. Localization
+The standard language for this Governance Framework (GF) is American English.
+
+## 1.4. Governing Authority 
+
+[The Tenure and Resource Stewardship Branch](https://www2.gov.bc.ca/gov/content/industry/natural-gas-oil/petroleum-natural-gas-tenure) is the governing authority and party responsible for developing, maintaining and implementing the Governance Framework (GF). 
+
+## 1.5. Administering Authority
+
+[Energy, Mines and Digital Trust (EMDT)](https://digital.gov.bc.ca/case-studies/emdt) is the Administering Authority on behalf of Tenure during the pilot phase of development.
+
+* 	**Name:** Kyle Robinson 
+* 	**Title:** Senior Strategic Advisor, Digital Trust Ecosystems
+* 	**Organization:** Brairtech Consulting Inc.
+* 	**Email:** kyle.robinson@brairtech.ca
+
+## 1.6. Purpose
+
+The purpose of this governance framework is to describe the rules/policies/procedures for verifiable credential exchanges involving the Tenure and Resource Stewardship Branch (TRSB) with the open global community. The purpose of the rules is to enable all actors to understand agreed upon standards, terminology and processes that allow the community to interact with TRSB in a trusted manner. This will help determine a governing framework and operating model for a global ecosystem that identifies how credentials can be issued, held, and verified. 
+
+## 1.7. Scope 
+The Tenure and Resource Stewardship Branch (TRSB) is a participant in an open ecosystem and the focus of this framework is to describe the process Tenure uses for digital exchanges on Indy networks.
+
+## 1.8. Objectives
+
+TBD
+
+## 1.9. Principles 
+
+TBD
+
+## 1.10. General Requirements
+TBD
+
+## 1.11. Revisions
+TBD
+
+## 1.12. Extensions
+
+There are no extensions to this Governance Framework.
+
+## 1.13. Schedule of Controlled Documents
+TBD
+
+# 2. Controlled Documents
+
+## 2.1. Glossary
+TBD
+
+## 2.2. Risk Assessment
+
+TBD
+
+## 2.3. Trust Assurance and Certification
+
+TBD
+
+## 2.4. Governance Requirements
+
+## 2.5. Business Requirements
+
+## 2.5.1. Establishment of Connection
+
+1. The Tenure and Resource Stewardship Branch (TRSB) MUST send an invitation to the other business entity via email to initiate the exchange of information. 
+2. The receiving party MUST use the invitation in order to establish a secure connection.
+
+## 2.5.2. Issuance of BC Petroleum & Natural Gas Title Credential 
+
```

Comment:
No credential request, TRSB just issues - but we could also discuss this process with TRSB